### PR TITLE
python310Packages.karton-core: 4.4.1 -> 5.0.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Description of changes
Update `karton` stack.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 185673` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>python310Packages.karton-asciimagic</li>
    <li>python310Packages.karton-autoit-ripper</li>
    <li>python310Packages.karton-classifier</li>
    <li>python310Packages.karton-config-extractor</li>
    <li>python310Packages.karton-core</li>
    <li>python310Packages.karton-dashboard</li>
    <li>python310Packages.karton-mwdb-reporter</li>
    <li>python310Packages.karton-yaramatcher</li>
    <li>python39Packages.karton-asciimagic</li>
    <li>python39Packages.karton-autoit-ripper</li>
    <li>python39Packages.karton-classifier</li>
    <li>python39Packages.karton-config-extractor</li>
    <li>python39Packages.karton-core</li>
    <li>python39Packages.karton-dashboard</li>
    <li>python39Packages.karton-mwdb-reporter</li>
    <li>python39Packages.karton-yaramatcher</li>
  </ul>
</details>

File: pkgs/development/python-modules/karton-classifier/default.nix [hunk 1]
```diff
@@ -27,17 +28,24 @@ buildPythonPackage rec {
...
 
-  checkInputs = [
-    pytestCheckHook
+  pythonImportsCheck = [
+    "karton.classifier"
   ];
 
-  pythonImportsCheck = [ "karton.classifier" ];
+  disabledTests = [
+    "test_process_archive_ace"
```
Comment (risicle) [inline pkgs/development/python-modules/karton-classifier/default.nix:46]:
```suggestion
    # tests expecting results from a different version of libmagic
    "test_process_archive_ace"
```

**Comment by risicle** [comment]
(apart from that, `nixpkgs-review` happy, macos 10.15 & nixos x86_64)

File: pkgs/development/python-modules/karton-core/default.nix [hunk 2]
```diff
@@ -8,16 +9,20 @@
...
     repo = "karton";
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-smgKrFexuL0bgt/1Ikm1tpSGPJNJm7Ko68iZn3AQw5E=";
+    hash = "sha256-0B2u2xnrGc3iQ8B9iAQ3fcovQQCPqdFsn5evgdDwg5M=";
   };
 
-  propagatedBuildInputs = [ minio redis ];
+  propagatedBuildInputs = [
+    boto3
+    minio
```
Comment (chivay) [inline pkgs/development/python-modules/karton-core/default.nix:23]: Is there any reason to keep `minio` in dependencies? It was removed and replaced by `boto3`

File: pkgs/development/python-modules/karton-classifier/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/karton-classifier/default.nix:47]: Thanks. Comment added.

File: pkgs/development/python-modules/karton-core/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/karton-core/default.nix]: Was just looking for new stuff and missed to remove it.

**Review by risicle — APPROVED** [review]
`nixpkgs-review` happy, macos 10.15, nixos x86_64 & aarch64

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 185673` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>python310Packages.karton-asciimagic</li>
    <li>python310Packages.karton-autoit-ripper</li>
    <li>python310Packages.karton-classifier</li>
    <li>python310Packages.karton-config-extractor</li>
    <li>python310Packages.karton-core</li>
    <li>python310Packages.karton-dashboard</li>
    <li>python310Packages.karton-mwdb-reporter</li>
    <li>python310Packages.karton-yaramatcher</li>
    <li>python39Packages.karton-asciimagic</li>
    <li>python39Packages.karton-autoit-ripper</li>
    <li>python39Packages.karton-classifier</li>
    <li>python39Packages.karton-config-extractor</li>
    <li>python39Packages.karton-core</li>
    <li>python39Packages.karton-dashboard</li>
    <li>python39Packages.karton-mwdb-reporter</li>
    <li>python39Packages.karton-yaramatcher</li>
  </ul>
</details>

**Pull request merged**